funfact/quantum/_qcircuit.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from numbers import Integral
+from funfact.lang._special import eye
+
+
+class Circuit():
+    '''A class to create quantum circuits and tensor expressions from them.'''
+    def __init__(self, nbqubits, offset=0):
+        if not (isinstance(nbqubits, Integral) and nbqubits > 0):
+            raise RuntimeError(
+                f'Number of qubits must be integer > 0, got {nbqubits}.'
+            )
+        if not (isinstance(offset, Integral) and offset >= 0):
+            raise RuntimeError(
+                f'Offset must be integer >= 0, got {offset}.'
+            )
+
+        self._nbqubits = nbqubits
+        self._offset = offset
+        self._qgates = []
+
+    @property
+    def nbqubits(self):
```

Comment:
What does this property do? Could we add a docstring for it?

Reply:
It is the number of qubits (or size) of the circuit. Included a doc string.